Source/AwsCommonRuntimeKit/crt/CRTError.swift

```diff
@@ -1,18 +1,30 @@
-//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-//  SPDX-License-Identifier: Apache-2.0.
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0.
 import AwsCCommon
 
-public struct CRTError: Error {
+extension CRTError {
+    public init(fromErrorCode errorCode: Int32) {
+        self = CRTError(rawValue: errorCode) ?? .UNKNOWN_ERROR_CODE
+    }
+
+    public init(fromErrorCode errorCode: Int) {
+        self = CRTError(rawValue: Int32(errorCode)) ?? .UNKNOWN_ERROR_CODE
+    }
 
-    public let errorCode: Int32
+    /// Use aws_last_error() as the error code
+    public init() {
+        self.init(fromErrorCode: aws_last_error())
```

Comment:
I'd rather this be explicit, like `CRTError.makeFromLastError()`

Swift suggests [Begin names of factory methods with “make"](https://www.swift.org/documentation/api-design-guidelines/)

Reply:
Thanks, updated.